test/test_runner.c

```diff
@@ -15,6 +15,19 @@ void CB2_TestRunner(void);
 EWRAM_DATA struct TestRunnerState gTestRunnerState;
 EWRAM_DATA struct FunctionTestRunnerState *gFunctionTestRunnerState;
 
+enum {
+    CURRENT_TEST_STATE_ESTIMATE,
+    CURRENT_TEST_STATE_RUN,
+};
+
+__attribute__((section(".persistent"))) static struct {
+    u32 address:28;
+    u32 state:1;
+} sCurrentTest = {
+    .address = 0,
+    .state = CURRENT_TEST_STATE_ESTIMATE,
+};
```

Comment:
```suggestion
} sCurrentTest = {0};
```
The above makes it sound like you can initialize the section. This could potentially just be uninitialized too?

Reply:
Fun fact you taught me this week: if we don't have an initializer agbcc will ignore our `section` attribute :D

But initializing with `{0}` is a good idea, I'll do that.